LDK/resources/web/LDK/Utils.js

```diff
@@ -538,15 +538,16 @@ LDK.Utils = new function(){
             window.location = LABKEY.ActionURL.buildURL('ldk', 'updateQuery', null, params);
         },
 
-        splitIds: function(subjectArray, unsorted, preserveDuplicates){
+        splitIds: function(subjectArray, unsorted, preserveDuplicates, preserveCase){
```

Comment:
It would make a whole lot more sense to invert this and make this 'caseInsensitive', which has the effect of keeping existing case-sensitive behavior, and letting new code opt-in to your extension. As proposed, your change is going to alter a lot of code outside of the codepaths you are trying to fix.

Reply:
I can change the name of the variable to caseInsensitive if that is more clear. I was trying to emphasize we do not want to force everything to lowercase when "preserveCase" or "caseInsensitive" is defined and true. This will not affect any other usage that hasn't opted in with caseInsensitive: true.